ingest/cell_metadata.py

```diff
@@ -182,14 +182,15 @@ def validate_header_keyword(self):
         if self.headers[0].casefold() == 'NAME'.casefold():
```

Comment:
I'm curious to know why you use casefold vs lower()?

Reply:
Had read that casefold handles more cases than lower but that's not relevant when we're expecting variants on "NAME" and "TYPE" so I've changed the code for better readability.
Reference about why/when casefold would be more appropriate: https://www.w3.org/International/wiki/Case_folding
code updated in `4563451`